### PR TITLE
C-667 - Home rebuild

### PR DESCRIPTION
**Pull request by gaspergrom**
# Changes proposed ✍️
- rebuild home page
- members & organizations query filters
  
- ### Screenshots (front-end changes only)
image
image

## Checklist ✅
- [x] Label appropriately with `Feature`, `Enhancement`, or `Bug`.
- [ ] Tests are passing.
- [ ] New backend functionality has been unit-tested.
- [ ] Environment variables have been updated:
  - [ ] Local frontend configuration: `frontend/.env.dist.local`, `frontend/.env.dist.composed`.
  - [ ] Local backend: `backend/.env.dist.local`, `backend/.env.dist.composed`.
  - [ ] [Configuration docs](https://docs.crowd.dev/docs/configuration) have been updated.
  - [ ] Team members only: update environment variables in override, staging and production env. files and trigger update config script.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.
- [ ] All changes have been tested in a staging site.
- [x] All changes are working locally running crowd.dev's Docker local environment.

### CODE REVIEW

File: frontend/src/shared/filter/components/filter-list.vue [hunk 1]
```diff
@@ -94,6 +94,7 @@ const filtersArray = computed(() =>
 )
 
 const handleFilterChange = (attribute) => {
+  console.log(attribute)
```
Comment (joanagmaia) [inline frontend/src/shared/filter/components/filter-list.vue:97]: Left behind 😄

File: scripts/cli [hunk 2]
```diff
@@ -495,6 +495,11 @@ do
             start
             exit;
         ;;
+        start-be)
+            declare -a INGORED_SERVICES=("premium-api" "premium-job-generator" "premium-python-worker" "community-help-center" "python-worker" "frontend" "discord-ws")
```
Comment (joanagmaia) [inline scripts/cli:499]: I think Uros removed these services `premium-api`, `premium-python-worker` and `premium-job-generator`, so no need to add those here

File: frontend/src/modules/member/pages/member-list-page.vue [hunk 3]
```diff
@@ -119,6 +120,41 @@ export default {
...
+      await this.updateFilterAttribute({
+        custom: false,
+        defaultOperator: 'gt',
+        defaultValue: joinedFrom,
+        expanded: false,
+        label: 'Joined date',
+        name: 'joinedAt',
+        operator: 'gt',
+        type: 'date',
+        value: joinedFrom
```
Comment (joanagmaia) [inline frontend/src/modules/member/pages/member-list-page.vue:135]: Should we validate the value type? To see if it matches what we want? Right now if I write `joinedFrom=string`, the application breaks

File: frontend/src/modules/dashboard/components/dashboard-count.vue [hunk 4]
```diff
@@ -162,30 +166,21 @@ export default {
...
         }
+        return `+100% (${currentScore})`
+      }
+      if (diff > 0) {
+        return `${Math.round(
+          (diff / previousScore) * 100
+        )}% (${diff})`
+      }
+      if (diff < 0) {
+        return `${Math.round(
```
Comment (joanagmaia) [inline frontend/src/modules/dashboard/components/dashboard-count.vue:181]: Both in positive and negative growth we shouldn't show the `-` or `+` sign. Nuno warned me about this as well in reports. For both scenarios this should be UI:
Screenshot 2023-02-22 at 15 35 59
Screenshot 2023-02-22 at 15 36 04

File: frontend/src/modules/dashboard/components/dashboard-active-integrations.vue [hunk 5]
```diff
@@ -1,74 +1,164 @@
...
-        effect="dark"
-        :content="platformDetails(active).name"
-        placement="top"
-      >
+  <section>
+    <h6 class="text-base leading-6 font-semibold pb-3">
+      Integrations
+    </h6>
+    <div v-if="activeIntegrations.length > 0" class="pb-1">
+      <div v-if="loading">
```
[the remark this answers is not in the document]
Comment (gaspergrom) [reply frontend/src/modules/dashboard/components/dashboard-active-integrations.vue]: yep this one was already resolved

File: frontend/src/modules/dashboard/components/dashboard-active-integrations.vue [hunk 6]
```diff
@@ -1,74 +1,164 @@
...
 import { mapActions, mapGetters } from 'vuex'
 import { CrowdIntegrations } from '@/integrations/integrations-config'
+import AppLoading from '@/shared/loading/loading-placeholder.vue'
 
 export default {
   name: 'AppDashboardIntegrations',
+  components: { AppLoading },
   data() {
     return {
+      loading: true,
```
[the remark this answers is not in the document]
Comment (gaspergrom) [reply frontend/src/modules/dashboard/components/dashboard-active-integrations.vue]: yeap already resolved

File: frontend/src/modules/dashboard/components/dashboard-filters.vue [hunk 7]
```diff
@@ -13,18 +16,20 @@
...
-        <span class="text-gray-900 pl-1">{{
-          getPlatformName
-        }}</span>
-        <i
-          class="ri-arrow-down-s-line text-base ml-1 transition transform"
-          :class="{ 'rotate-180': platformDropdownOpen }"
-        ></i>
-      </div>
+      <el-button
+        class="custom-btn bg-white"
```
[the remark this answers is not in the document]
Comment (gaspergrom) [reply frontend/src/modules/dashboard/components/dashboard-filters.vue]: Fixed

**Review by joanagmaia — APPROVED** [review]
LGTM! Thanks for taking care of all the issues

**Pull request merged**